src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java

```diff
@@ -276,7 +278,7 @@ private void activeSpeakerChanged(long ssrc)
     {
         Conference conference = getConference();
 
-        if ((conference != null) && !conference.isExpired())
+        if (conference != null)
```

Comment:
I'm sorry, please disregard this, I just noticed that you're taking care of that in the `getConference` method.